std/math.d

```diff
@@ -4066,11 +4066,11 @@ private:
         // ARM FPSCR is a 32bit register
         enum : int
         {
-            INEXACT_MASK   = 0x1000,
-            UNDERFLOW_MASK = 0x0800,
-            OVERFLOW_MASK  = 0x0400,
-            DIVBYZERO_MASK = 0x0200,
-            INVALID_MASK   = 0x0100
+            INEXACT_MASK   = 0x10,
+            UNDERFLOW_MASK = 0x08,
+            OVERFLOW_MASK  = 0x04,
+            DIVBYZERO_MASK = 0x02,
+            INVALID_MASK   = 0x01
```

Comment:
You see, what you are doing here is setting them to the bits representing exceptions.  What the name implies is the mask.

This change is wrong.